build/packages-template/bbb-export-annotations/build.sh

```diff
@@ -26,6 +26,11 @@ popd
 mkdir -p staging/usr/lib/systemd/system
 cp bbb-export-annotations.service staging/usr/lib/systemd/system
 
+#
+# Install fonts
+mv fonts/* /usr/local/share/fonts/
+fc-cache -f
```

Comment:
fs-cache -f should go in the post-install script, I believe.

Reply:
Done, it's in after-install.sh now.